app/services/description_import.rb

```diff
@@ -71,7 +71,10 @@ def key?(what, key)
 
   def compact_params(params)
     params.each do |key, value|
-      params[key].compact! if value.is_a?(Array)
+      next unless value.is_a?(Array)
```

Comment:
Won't this kill the recursion? I'm not sure the test shows that this works on nested arrays that have nils.

Reply:
Can you test with the data you display in the PR?

Reply:
i can make that change. Just to point out, this does show the functionality. If I remove the compact_params method and run this test I get a "Cannot insert nil..." error - so I believe this is properly exercising the code, but I can make deeper test data, sure.

Reply:
@jcoyne ok, this is now testing with the data from the comment above.